src/components/Visualization.tsx

```diff
@@ -25,8 +25,6 @@ const Visualization = () => {
     zoom: 1.2,
   };
   const [isPanelExpanded, setIsPanelExpanded] = useState(false);
-  const [, setReactFlowInstance] = useState(null);
```

Comment:
@kahboom is this needed? I think that it forces a render but I don't know if there another use case for it :thinking:

Reply:
confession time--I have no idea. I think that might have been from an older version of React Flow, but I can't be sure. 😅  We can remove it, just be sure to check rendering still happens as needed with the basic actions (for example replacing from big catalog, deleting steps, editing config, etc.)

Reply:
Mmm, I think that you're right, I checked [here](https://reactflow.dev/docs/guides/migrate-to-v10/#4-onload---oninit) and it seems that it's also deprecated.